plugin/ToolBar.qml

```diff
@@ -77,6 +77,7 @@ PanelBackground {
         spacing: Theme.paddingLarge
         anchors.verticalCenter: parent.verticalCenter
```

Comment:
This is perhaps now unnecessary if height matches?

Reply:
I agree, removed.